config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
   devise_for :users
   root controller: :main, action: :index
+  resources :posts, only: [:new, :create, :index, :show, :edit, :destroy]
```

Comment:
Hmm....  why `only` if you include ALL actions?

Reply:
done